applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -230,6 +230,15 @@ config WALKIE_TALKIE_DEMO
 	  The walkie talkie demo will set up a bidirectional stream using PDM
 	  microphones on each side.
 
+
+# Choice on stereo vs mono streams.
```

Comment:
I dont think this comment is needed, since the config is described in "help".
Should also remove extra newline

Reply:
Good catch, that was just a memo to self.